### PR TITLE
Allow emission of i64::MIN integer literal

### PR DESCRIPTION
**Pull request by jethrogb**
This change is [Reviewable](https://reviewable.io/reviews/crabtw/rust-bindgen/369)

### CODE REVIEW

**Comment by jethrogb** [comment]
Take care when merging together with #371

File: src/gen.rs [hunk 1]
```diff
@@ -957,9 +957,17 @@ fn cunion_to_rs(ctx: &mut GenCtx,
     items
 }
 
+fn i64_abs(i: i64) -> u64 {
+    if i<0 {
+        i.wrapping_neg() as u64
+    } else {
+        i as u64
+    }
+}
```
Comment (Yamakaky) [inline src/gen.rs]: I think `i as u64` is sufficient?

Comment (jethrogb) [inline src/gen.rs]: That would turn `-1` into `18446744073709551615`, etc.

**Comment by jethrogb** [comment]
Rebased

File: src/gen.rs (hunk 1 above)
Comment (Yamakaky) [inline src/gen.rs]: Oh sorry, I thought you wanted the two complement value.

**Comment by Yamakaky** [comment]
could you update the changelog please?

**Comment by jethrogb** [comment]
Done.

**Pull request merged**